sport-moment-nft/contracts/EPL.cdc

```diff
@@ -0,0 +1,1124 @@
+/*
+    Contract for EPL sports moments & metadata
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import FungibleToken from 0x{{.FungibleTokenAddress}}
+import NonFungibleToken from 0x{{.NonFungibleTokenAddress}}
+import MetadataViews from 0x{{.MetadataViewsAddress}}
+
+/// The EPL NFT and metadata contract
+//
+pub contract EPL: NonFungibleToken {
```

Comment:
Maybe this should be called `EnglishPremeirLeague`?

Reply:
Done.